acqua-backend/app/Http/Controllers/TicketController.php

```diff
@@ -19,13 +20,76 @@ public function index()
         return Ticket::orderBy('created_at', 'desc')->paginate(15);
     }
 
+    public function generateReport() : JsonResponse
+    {
+
+        try {
+            $inicioFechaConsulta = request('fecha_inicio') ?? null;
+            $finFechaConsulta = request('fecha_fin') ?? null;
+
+            if (empty($inicioFechaConsulta) && empty($finFechaConsulta)) {
+                // Fecha Inicio y Final no Proporcinadas
+                $inicioFechaConsulta = Carbon::now()->startOfDay();
+                $finFechaConsulta = Carbon::now()->endOfDay();
+
+            } else {
+                /// Fechas Proporcinadas
+                $inicioFechaConsulta = Carbon::createFromFormat('Y-m-d H:i:s', request('fecha_inicio'))->startOfDay();
+                $finFechaConsulta = Carbon::createFromFormat('Y-m-d H:i:s', request('fecha_fin'))->endOfDay();
+            }
+
+            // * Consulta para metodo de pago "Contado" Pagado
+            $montoTicketsPagadosAContado = Ticket::where('restante', 0)
+                ->where('tipo_credito', 'contado')
+                ->where('vencido', false)
```

Comment:
->where()->where()->where() anida el query final? O sobreescribe al primero?

Reply:
No camo tal, se están aplicando múltiples condiciones para filtrar los registros que se sumarán (una especie de AND en a los registros que si los pasa los sumara)